packages/luals-annotations/src/index.ts

```diff
@@ -0,0 +1,47 @@
+import { classes, functions, matchers } from "./json-docs";
+import AnnotationBuilder from "./annotationBuilder";
+import * as fs from "fs";
+
+const document = new AnnotationBuilder();
+document.add("---@meta");
+
+classes.forEach((classDef) => {
+	document.withClass(classDef.name, classDef.description, false, () => {
+		classDef.fields?.forEach((field) => document.addField(field));
+		document.addClassDeclaration();
+		classDef.methods?.forEach((method) => document.addFunction(method));
+	});
+});
+
+functions.forEach((func) => {
+	document.addFunction(func);
+});
+
+document.withClass("lest.Matchers", "Matchers for expect()", true, () => {
+	document.addField({
+		name: "never",
+		type: "lest.InverseMatchers",
+		description: "Inverse matchers",
+	});
+
+	document.addClassDeclaration();
+	matchers.forEach((matcher) => document.addFunction(matcher));
+});
+
+document.withClass("lest.InverseMatchers", "Inverse matchers", true, () => {
+	document.addClassDeclaration();
+	matchers.forEach((matcher) => document.addFunction(matcher));
+});
+
+if (process.env["npm_config_debug"]) {
```

Comment:
What's this?

Reply:
Forgot to comment that, but npm has a feature where any double dash argument (like `--debug`) gets passed onto the env as npm_config_X.